src/transformers/models/levit/convert_levit_timm_to_pytorch.py

```diff
@@ -167,13 +167,14 @@ def convert_weights_and_push(save_directory: Path, model_name: str = None, push_
         required=False,
         help="Path to the output PyTorch model directory.",
     )
+    parser.add_argument("--push_to_hub", action="store_true", help="Push model and feature extractor to the hub")
     parser.add_argument(
-        "--push_to_hub",
-        default=True,
-        type=bool,
-        required=False,
-        help="If True, push model and feature extractor to the hub.",
+        "--no-push_to_hub",
+        dest="push_to_hub",
+        action="store_false",
+        help="Do not push model and feature extractor to the hub",
     )
+    parser.set_defaults(push_to_hub=True)
```

Comment:
(nit) I'm not using this in any conversion script

Reply:
@NielsRogge  removed.